packages/blade/src/utils/useBreakpoint/useBreakpoint.ts

```diff
@@ -0,0 +1,142 @@
+import { useEffect, useState, useCallback, useMemo } from 'react';
+import type { Breakpoints } from '../../tokens/global';
+
+type DeviceType = 'desktop' | 'mobile';
```

Comment:
Let's use an object here. 

```ts
const device = {
  desktop: "desktop",
  mobile: "mobile",
} as const; 
type DeviceType = keyof typeof device;
```
And remove all hardcoded string "mobile", "desktop" magic values.

Reply:
what will we gain by doing that?

Reply:
So that we don't have scattered string values for "mobile" & "desktop". 

<img width="533" alt="Screenshot 2022-03-14 at 11 49 04 AM" src="https://user-images.githubusercontent.com/35374649/158115954-4064ea11-69b7-431a-96c2-01fb7221d387.png">